### PR TITLE
Add mediation policy using inline content.

### PR DESCRIPTION
**Pull request by Meruja**
Fixes https://github.com/wso2/product-apim/issues/6851

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java [hunk 1]
```diff
@@ -2268,6 +2278,7 @@ public Response apisApiIdMediationPoliciesPost(String type, String apiId, InputS
         return null;
     }
 
+
```
Comment (kavishkafernando) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java]: Shall we remove this extra line?

Comment (Meruja) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java]: Removed in 7e871d9

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java [hunk 2]
```diff
@@ -3852,6 +3860,17 @@ public String getMediationNameFromConfig(String config) {
         return null;
     }
 
+    /**
+     * Check the existence of the mediation policy
```
Comment (rmsamitha) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java:3864]: Needs empty line after the method description

Comment (Meruja) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java:3864]: Fixed with f81bbb7

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java [hunk 3]
```diff
@@ -3852,6 +3860,17 @@ public String getMediationNameFromConfig(String config) {
...
+     * @param mediationResourcePath mediation config content
+     *
+     */
+    public void checkMediationPolicy(APIProvider apiProvider,String mediationResourcePath) throws APIManagementException {
```
Comment (tharikaGitHub) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java:3868]: Let's fix the formatting issue in this method. Need a space after the comma in the list of parameters.

Comment (Meruja) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java:3868]: Fix with 5872157

**Pull request merged**